docs/guides/git-vs-composer-install.md

```diff
@@ -0,0 +1,84 @@
+#Git vs Composer Installation of Magento with MFTF
```

Comment:
Any place where you use `# Headings` you need to include a space between the `#` and the heading itself

Reply:
fixed